### ISSUE TITLE
Misleading Line In Getting Started

### ISSUE THREAD

**Issue by wiverson** [openjfx/openjfx-docs#196]
>Non-modular application
>
>Since Java 9, applications should be modular, and distributed with tools like jlink.

I would suggest that this entire section be reworked to deemphasize trying to make the end user application modular. I tried to make this work and ran into nothing but problems.

I spent a long, long time trying to make my app[s] fully modular and it was essentially impossible for anything but a trivial Hello World app. As soon as I brought in any dependencies it turned into a nightmare.

Here is a very simple, clean Maven template for building a JavaFX app, including GitHub Actions for creating the installers.

https://github.com/wiverson/maven-jpackage-template

Feel free to grab any/all of it for the JavaFX documentation.

**Comment by antmordel** [comment]
I agree with this, I spent long hours trying to make a modular app to work. With no success. I wasn't lucky enough to see this issue at the first try.

**Comment by abhinayagarwal** [comment]
This part of the documentation should get an update, especially now that `jpackage` is out of incubation and a part of jdk tools.

I will have a look at it, unless someone beats me to it :)

**Comment by wiverson** [comment]
I would be happy to send in some updates for docs. Is a PR the preferred process?

**Comment by abhinayagarwal** [comment]
@wiverson A PR would be great!
